python_modules/dagster/dagster/core/execution.py

```diff
@@ -186,9 +186,23 @@ def output_expectations(self):
     @property
     def success(self):
         '''Whether the solid execution was successful'''
+        any_success = False
```

Comment:
may be worth checking out python functional idioms like `any` if you are interested